src/main/java/dan200/computercraft/ComputerCraft.java

```diff
@@ -277,6 +281,9 @@ public void preInit( FMLPreInitializationEvent event )
         Config.turtlesCanPush = Config.config.get( Configuration.CATEGORY_GENERAL, "turtlesCanPush", turtlesCanPush );
         Config.turtlesCanPush.setComment( "If set to true, Turtles will push entities out of the way instead of stopping if there is space to do so" );
 
+        Config.minTimeBetweenSounds = Config.config.get( Configuration.CATEGORY_GENERAL, "minTimeBetweenSounds", minTimeBetweenSounds);
+        Config.minTimeBetweenSounds.setComment("The minimum time in between calls to sound.play on one computer in milliseconds" );
```

Comment:
I had added the `setMinValue`. I don't understand the language key: won't this snippet do it - 
```java
for (Property property : Config.config.getCategory( Configuration.CATEGORY_GENERAL ).getOrderedValues())
        {
            property.setLanguageKey( "gui.computercraft:config." + CaseFormat.LOWER_CAMEL.to( CaseFormat.LOWER_UNDERSCORE, property.getName() ) );
        }
```